.travis.yml

```diff
@@ -3,33 +3,36 @@ language: python
 matrix:
   include: 
     - python: "2.7"
-      env: DEPS="numpy=1.7.1 scipy=0.13.0 nose matplotlib=1.3 pandas=0.13.0 scikit-image=0.9 pyyaml pytables"
+      env: DEPS="numpy=1.7.1 scipy=0.13.0 matplotlib=1.3 pandas=0.13.0 scikit-image=0.9 pyyaml pytables"
     - python: "2.7"
-      env: DEPS="numpy=1.7.1 scipy=0.13.0 nose matplotlib=1.3 pandas=0.13.0 scikit-image=0.9 pyyaml numba=0.11 pytables"
+      env: DEPS="numpy=1.7.1 scipy=0.13.0 matplotlib=1.3 pandas=0.13.0 scikit-image=0.9 pyyaml numba=0.11 pytables"
     - python: "2.7"
-      env: DEPS="numpy=1.8 scipy nose matplotlib=1.3 pandas=0.13.0 scikit-image=0.10.1 pyyaml numba=0.12 pytables"
+      env: DEPS="numpy=1.8 scipy matplotlib=1.3 pandas=0.13.0 scikit-image=0.10.1 pyyaml numba=0.12 pytables"
     - python: "2.7"
-      env: DEPS="numpy=1.8 scipy=0.14.0 nose matplotlib=1.3 pandas=0.13.0 scikit-image=0.10.1 pyyaml numba=0.12.2 pytables"
+      env: DEPS="numpy=1.8 scipy=0.14.0 matplotlib=1.3 pandas=0.13.0 scikit-image=0.10.1 pyyaml numba=0.12.2 pytables"
     - python: "3.4"
-      env: DEPS="numpy=1.8 scipy=0.14.0 nose matplotlib=1.3 pandas=0.14.0 scikit-image=0.10.1 pyyaml pytables"
+      env: DEPS="numpy=1.8 scipy=0.14.0 matplotlib=1.3 pandas=0.14.0 scikit-image=0.10.1 pyyaml pytables"
 
 install:
-  - conda install --yes -c conda conda-env
-  - conda create -n testenv --yes $DEPS pip python=$TRAVIS_PYTHON_VERSION
+  # See:
+  # https://groups.google.com/a/continuum.io/forum/#!topic/conda/RSFENqovnro
+  # - conda update --yes --no-deps conda
+  # OK, this used to *fix* the build, but now it *breaks* the build.
+  # If you're reading this, good luck out there. I'm not sure what to tell you.
```

Comment:
I bet it is the lack of conda-env that does _not_ get pulled in now due to the `--no-deps` where as before there some some strange race condition.  Does doing _both_ work?